cmd/ndm_daemonset/probe/usedbyprobe.go

```diff
@@ -149,6 +150,18 @@ func (sp *usedbyProbe) FillBlockDeviceDetails(blockDevice *blockdevice.BlockDevi
 		}
 	}
 
+	// checking for lvm localPV
+	usedByProbe := newUsedByProbe(blockDevice.DevPath)
+	// check for LVM file system
+	fstype := usedByProbe.BlkidIdentifier.GetOnDiskFileSystem()
```

Comment:
Is this a reliable way to check if the device is  in use by LVM localPV. Because nodes having LVM setup will also have the same lvm filesystem on the disks.

we need to check if this method helps us to identify between lvm localPV and LVM that has been setup by user.

Reply:
This same issue is there for cstor and zfs -localpv. To identify that, we check if the device is exclusively locked. which happens only if kernel zfs is using the disk.

Reply:
> Because nodes having LVM setup will also have the same lvm filesystem on the disks.

By this you mean, disks that are used manually by LVM utils?